cernopendata/modules/fixtures/data/docs/cod-privacy-policy/cod-privacy-policy.md

```diff
@@ -1 +1,61 @@
+
+### Personal Data we process
+
+The personal data we have, and how it's used:
+
+Personal Data | Purpose | Basis | Source
+------------- | ------- | ----- | -------
+Your access logs (IP address, visited URLs on CERN Open Data, and corresponding timestamp)  | User support, website debugging, security auditing and to produce statistics  | Legitimate interest of CERN | Automatically transferred by your web browser
+
+#
+**Description of legal basis for processing of Personal Data by CERN Open Data**
+
+-    **Contract:** To fulfil a contractual relationship with the individual, or in preparation for a contract with the individual
+-    **Legal Obligation:** To comply with a legal obligation of CERN.
+-    **Consent:** By having received and recorded consent from the individual.
+-    **Legitimate interest of CERN:** In the legitimate interests of CERN supporting the professional activities of the individual or their security and safety.
+
+### Personal Data we keep
+
+The personal data we store, for how long and why:
+
+Personal Data | Retention Period <sup>1</sup> | Purpose
+------------- | ----------------------------- | --------
+Your access logs (IP address, visited URLs on CERN Open Data, and corresponding timestamp) | Maximum 13 months from date of action | User support, website debugging, security auditing and to produce statistics
+
+### Who at CERN has access
+
+In addition to yourself, personal data collected by CERN Open Data is accessible by the following services, teams or individuals at CERN:
+
+Personal Data | Who | Purpose
+------------- |---- | --------
+Your access logs (IP address, visited URLs on CERN Open Data, and corresponding timestamp) | CERN Open Data administrators and CERN Services administrators (OpenShift service) | User support and service operations
+
+#
+For more detailed information about personal data and privacy please refer to the [Data Privacy web site](https://privacy.web.cern.ch/).
+
+For questions regarding this Privacy Notice, please contact [this Service](https://cern.service-now.com/service-portal?id=sc_cat_item&name=general-help-rqf&fe=service-desk&short_description=Question%20about%20Privacy%20Notice%20%22CERN%20Open%20Data%22%20of%20Service:%20Open%20Science%20Infrastructure&optional_functional_element=e93478690a0a8c0a009abf9422a74c71&comments=%0A%0ANote%20for%20the%20Service%20Desk:%20if%20possible,%20please%20assign%20this%20ticket%20to%20the%20Service%20Element:%20%22Open%20Science%20Infrastructure%22.).
+
+For questions regarding personal data and privacy please contact the [Office of Data Privacy](https://privacy.web.cern.ch/office-data-privacy-odp).
+
+To request to exercise data subject rights please fill and submit the following [online form](https://cern.service-now.com/service-portal?id=sc_cat_item&name=data-subject-rights&se=Data-Privacy).
+
+This Privacy Notice is subject to revision.
+
+**Last revision: 09-10-2020 11:27:51**
```

Comment:
We are using YYYY-MM-DD format usually on the site, so please switch to that.

Also dunno about it being bold; it is too much visible, while it is sort of ancillary information.  You can simply use regular font, or else switch to italic.